Doc/using/cmdline.rst

```diff
@@ -713,6 +713,35 @@ conflict.
 
    .. versionadded:: 3.6
 
+
+.. envvar:: PYTHONCOERCECLOCALE
+
+   If set to a non-empty string, causes the main Python command line application
+   to skip coercing the legacy ASCII-based C locale to a more capable UTF-8
+   based alternative. Note that this setting is checked even when the
+   :option:`-E` or :option:`-I` options are used, as it is handled prior to
+   the processing of command line options.
```

Comment:
Fixed in https://github.com/python/cpython/pull/659/commits/1c3a2706ef276ca66f676218395b9d792ef3ca0e